### ISSUE TITLE
Option for other direction for horizontal scrolling

### ISSUE THREAD

**Issue by jenuk** [ahrm/sioyek#637]
For me sioyek scrolls in the opposite direction when scrolling horizontally compared to the other pdf viewers I have. 

I have found issue #39, I don't know if this a regression (I believe that the code from the commit that fixed that issue has changed since then) or if the other person just preferred the other way around. In both cases, the option to switch around scrolling would be nice.

I am on ubuntu 20.04, if that matters.

**Comment by ahrm** [comment]
Added a `inverted_horizontal_scrolling` config option in c5c5a9bbcdbc1ab7d8ab06fe023d03f91c6a61f7.

**Comment by jenuk** [comment]
Hi, sorry for not getting back to you earlier, had problems with building it myself, and was only just now able to test your changes. Thank you!

However, I think I didn't expressed my request clearly: My laptop's touchpad can scroll horizontal natively and I was talking about that (I was not aware you could hold shift to scroll horizontally). So, I want to propose to change (that is in the case of no shift nor control key being pressed):

https://github.com/ahrm/sioyek/blob/426b7886907a4e527bb6db47c00884cce10b1b6d/pdf_viewer/main_widget.cpp#L1794-L1801

to

```c++
            float inverse_factor = INVERTED_HORIZONTAL_SCROLLING ? -1.0f : 1.0f;
            if (wevent->angleDelta().x() > 0) {
                move_horizontal(-72.0f * horizontal_move_amount * num_repeats_f * inverse_factor);
                return;
            }
            if (wevent->angleDelta().x() < 0) {
                move_horizontal(72.0f * horizontal_move_amount * num_repeats_f * inverse_factor);
                return;
            }

```

Thank you again for your time!

**Comment by ahrm** [comment]
Should be fixed in 90d50c08b9681e30c9823f4bfab6eb001202a6cd.
